deploy/requirements.txt

```diff
@@ -50,15 +50,15 @@ protobuf>=3.15.5
 psycopg2-binary==2.9.6
 pydantic>=1.7.3,<=1.11.0
 python_dateutil==2.8.1
-pytz ~=2019.3
+pytz ~=2022.1
 requests>=2.22.0,<=2.28
 rq==1.10.1
 rules==2.2
 ujson>=3.0.0
 xmljson==0.2.0
 colorama>=0.4.4
 boxing>=0.1.4
-redis>=4.2.0
+redis~=3.5
```

Comment:
why do we need to touch it? it can affect users who install LS ML backend in the same env as LS (they like to do it).

Reply:
This redis requirment shouldn't confiligt with ML backend either. https://github.com/heartexlabs/label-studio-ml-backend/blob/cb98234e3afb6bae2c8ca8a35941a73aceaf3ef8/requirements.txt#L13 
`redis<4.6.0` and `redis>=4.2.0` don't conflict.
Is there another repo's dependencies that we need to be thinking about?